pkg/util/provider/drain/drain.go

```diff
@@ -881,7 +920,7 @@ func (o *Options) waitForDetach(ctx context.Context, podVolumeInfo PodVolumeInfo
 		}
 	}
 
-	klog.V(4).Infof("Detached volumes:%s from node %q", podVolumeInfo.volumeList, nodeName)
+	klog.V(3).Infof("Detached volumes %v from node %q", podVolumeInfo.volumes, nodeName)
```

Comment:
Won't this be too verbose at at the default log level 3 for nodes with large number of volumes ?

Reply:
I raised the log level to 3 here to be consistent with `waitForReattach`. If we are concerned about too verbose logs, we should also set the corresponding logs in `waitForReattach` to 3 as well, as they are logged for the same list of volumes. I just thought it didn't make sense to log the list in one function but not the other.